### DISCUSSION TITLE
Best practice for device names?  (re: location + name, Home Assistant, etc)

### DISCUSSION THREAD

**Discussion by sofakng** [zwave-js/zwave-js-ui#2671] [General]
Does anybody have any suggestions or best practices for naming Z-Wave devices in Z-Wave JS?

For example, let's say I have a Z-Wave ceiling fan switch in my living room.  Typically I've been using "Ceiling Fan" for the name and "Living Room" for the location.

This works fairly well but when I integrate with Home Assistant it only uses the device name so this would become "light.ceiling_fan" and causes confusion because a different room's ceiling fan is simply "light.ceiling_fan2".

Should I name the actual Z-Wave device "Living Room - Ceiling Fan" ?  How does everybody else name their Z-Wave devices?

**Comment by robertsLando** [comment]
I cannot help on the hass side but I usually set both name and location as you are doing to identify nodes. @kpine any suggestions for the hass side?
